gradle/dependencies.gradle

```diff
@@ -31,6 +31,7 @@ ext.dep = [
     recyclerView            : "com.android.support:recyclerview-v7:$versions.supportLibVersion",
     compiletesting          : 'com.google.testing.compile:compile-testing:0.15',
     cache                   : "com.nytimes.android:cache:$versions.cacheVersion",
+    kotlinpoet              : 'com.squareup:kotlinpoet:1.0.0-RC2',
```

Comment:
I took so long to review that 1.0.0 came out :-D

Reply:
it lasts so long that already 1.0.1 xD